deps/childprocess.lua

```diff
@@ -164,4 +164,71 @@ local function spawn(command, args, options)
   return em
 end
 
+local function exec(command, args, options, callback)
+  local file
+  local stringArgs = command..table.concat(args, ' ')
```

Comment:
Yep, `execFile` is lower level and is used by `exec`.  The important thing I wanted to note is that `exec` doesn't take args, but does inject `sh` or `cmd` while `execFile` is just spawn with output buffered.